ui/shared/components/panel/view-pedigree-image/PedigreeImagePanel.jsx

```diff
@@ -75,7 +75,7 @@ const EDIT_INDIVIDUAL_FIELDS = [
 
 const INDIVIDUAL_FIELD_MAP = {
   name: 'individualGuid',
-  display_name: 'displayName',
+  label: 'displayName',
```

Comment:
Is it okay to keep the value `displayName` for the field `label`?

Reply:
yes, because thats literally the entire purpose of this PR? I'm not sure what you are asking

Reply:
I see it is fine now. I thought the `displayName` field of the Individual also might be changed to `label` as well.